### PR TITLE
Add media player to cast app

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Adds a media player to our cast app

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: cast/src/receiver/entrypoint.ts [hunk 1]
```diff
@@ -30,13 +75,57 @@ options.uiConfig = new cast.framework.ui.UiConfig();
 // @ts-ignore
 options.uiConfig.touchScreenOptimizedApp = true;
 
+castContext.setInactivityTimeout(86400); // 1 day
+
 castContext.addCustomMessageListener(
   CAST_NS,
   // @ts-ignore
   (ev: ReceivedMessage<HassMessage>) => {
+    if (
```
Comment (balloob) [inline cast/src/receiver/entrypoint.ts:84]: Could we add some comments what we're doing here

File: cast/src/receiver/entrypoint.ts [hunk 2]
```diff
@@ -30,13 +75,57 @@ options.uiConfig = new cast.framework.ui.UiConfig();
...
+);
+
+const playerManager = castContext.getPlayerManager();
+
+playerManager.setMessageInterceptor(
+  cast.framework.messages.MessageType.LOAD,
+  (loadRequestData) => {
+    showMediaPlayer();
+    const media = loadRequestData.media;
+    if (media.entity) {
```
Comment (balloob) [inline cast/src/receiver/entrypoint.ts:105]:
```suggestion
    // Special handling if it came from Google Assistant
    if (media.entity) {
```

**Review by balloob — APPROVED** [review]
🎉

**Comment by hunterjm** [comment]
You’re the man @bramkragten!

**Pull request merged**